charts/openmetadata/templates/check-db-migrations-job-hook.yaml

```diff
@@ -7,6 +7,24 @@ metadata:
     "helm.sh/hook": pre-upgrade 
     "helm.sh/hook-weight": "-1" # we use a smaller weight so it's created before the job  
     "helm.sh/hook-delete-policy": hook-succeeded, hook-failed
+    {{- if .Values.annotations }}
+    {{- range $key, $val := fromYaml .Values.annotations }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end}}
+    {{- end}}
+  labels:
+    app.kubernetes.io/name: '{{ include "OpenMetadata.name" .}}'
```

Comment:
I would recommend using [helper class](https://github.com/open-metadata/openmetadata-helm-charts/blob/fd71bcc9eab12eebc13296f7b3f4a3a1f98dd392/charts/openmetadata/templates/_helpers.tpl) to define labels starting with `app.kubernetes.io/*`!

Reply:
used helper class to define labels starting with app.kubernetes.io/*